### DISCUSSION TITLE
HowTo: Is there a way to terminate a long running "sendkey" commands in between while it is sending key strokes?

### DISCUSSION THREAD

**Discussion by vbabaria** [socsieng/sendkeys#78] [General]
Nice utility, I am curious to know if there is a way to terminate a long running "sendkey" commands in between while it is sending key strokes? thanks for contributing to open source community.

**Comment by socsieng** [comment]
Yes, you can now do it by specifying the `--terminate-command` option. It follows the same convention as the key presses and modifier keys. Strip out the angle brackets.

Example, to use `control` + `c`:

- Instead of: `<c:c:control>`,
- Use: `c:control`
